docs/replication-replica.md

```diff
@@ -0,0 +1,51 @@
+# Configuring cross-site replication on Replica instances
+
+When the Operator creates a new cluster, a lot of things are happening, such as
+electing the Primary, generating certificates, and picking specific names. This
+should not happen if we want the Operator to run the _Replica_ site, so first
+of all the cluster should be put into unmanaged state by setting the
+`unmanaged` key in the `deploy/cr.yaml` configuration file to true. Also you
+should set `updateStrategy` key to `OnDelete` and `backup.enabled` to
+`false`, because [Smart Updates](update.md#operator-update-smartupdates) and
```

Comment:
the link goes to operator updates, SmartUpdate is not related to operator updates but it's related to cluster updates